tests/driver_mrf24j40/Makefile

```diff
@@ -2,6 +2,27 @@ INCLUDES += -I$(APPDIR)
 
 include ../Makefile.tests_common
 
+TEST_ON_CI_WHITELIST += adafruit-clue \
```

Comment:
better add a note that this is due to missing Kconfig support, not something inherently special about these boards

Reply:
hmmm I will take a closer look because it seems like it should be there... if the nrf52 common stuff is at least.

I can do a followup PR if it is something simple.